addons/60fps/addon.json

```diff
@@ -1,10 +1,10 @@
 {
-  "name": "Alt+GreenFlag 60FPS player mode",
-  "description": "Alt+Click the green flag to toggle 60FPS.",
+  "name": "Custom FPS player mode",
+  "description": "Alt+Click the green flag to toggle Custom FPS.",
   "info": [
     {
       "type": "notice",
-      "text": "Projects will run twice as fast at 60FPS, because they typically run at 30FPS.",
+      "text": "Projects will run at a different speed when in use. Keep in mind that projects typically run at 30 fps",
```

Comment:
Notices should end with a dot, like descriptions. I would change it to "Enabling the custom FPS mode changes the project speed. Projects typically run at 30 FPS."